operator/config/istio/gataway.yaml

```diff
@@ -2,7 +2,7 @@
 apiVersion: networking.istio.io/v1beta1
 kind: Gateway
 metadata:
-  name: lifecycle-manager-kyma-gateway
+  name: kyma-gateway
```

Comment:
This name has to be `lifecycle-manager-kyma-gateway` until watcher can dynamically using existing gateway, please check the patch I introduced in this PR https://github.com/kyma-project/lifecycle-manager/pull/262/files#diff-6ff3fbcd21a686f5d672eb6d639fb2b9c018e668eb3dd2588589f09465659186R6

If you change to this way, in KCP deployment, this gateway will rename to `klm-kyma-gateway`

Reply:
The patch you proposed doesn't help the `default` Kustomize build unfortunately

Reply:
The end result comes out to be` lifecycle-manager-lifecycle-manager-kyma-gateway` !!